User.py

```diff
@@ -0,0 +1,43 @@
+"""User module contains all of the components necessary for the User
```

Comment:
you do really good commit comments and subcomments, keep that up as it's noticed. you may want to break up your initial commits into logical chunks to emphasize this.